modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogSortedIndexDescriptor.java

```diff
@@ -45,6 +45,7 @@ public class CatalogSortedIndexDescriptor extends CatalogIndexDescriptor {
      * @param tableId Id of the table index belongs to.
      * @param unique Unique flag.
      * @param columns A list of columns descriptors.
+     *
```

Comment:
```suggestion
```

Reply:
Why? I like when `@param` tags are grouped together and separated

Reply:
I don't insist